pkg/v1/providers/ytt/02_addons/management-packages/add_management_package_repo.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       image: #@ "{}/{}:{}".format(get_image_repo_for_component(management_package_repo_bundle), management_package_repo_bundle.imagePath, management_package_repo_bundle.tag)
 #@ end
 
-#@ if data.values.TKG_CLUSTER_ROLE != "workload" and data.values.PROVIDER_TYPE != "tkg-service-vsphere":
+#@ if data.values.TKG_CLUSTER_ROLE != "workload" and data.values.PROVIDER_TYPE != "tkg-service-vsphere" and not data.values.FEATURE_FLAG_PACKAGE_BASED_LCM:
```

Comment:
With ytt overlays, does `not data.values.FEATURE_FLAG_PACKAGE_BASED_LCM` mean `FEATURE_FLAG_PACKAGE_BASED_LCM: false` or can it also mean that the `FEATURE_FLAG_PACKAGE_BASED_LCM` is empty?